cmd/otelcontribcol/builder-config.yaml

```diff
@@ -66,6 +66,7 @@ exporters:
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/lokiexporter v0.80.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/mezmoexporter v0.80.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opencensusexporter v0.80.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opensearchexporter v0.80.0
```

Comment:
I don't believe this should be added here at this time.  Per the [new component contribution guidelines](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md):

>* **Last PR** should mark the new component as `Alpha` stability and add it to the `cmd/otelcontribcol`
  binary by updating the `cmd/otelcontribcol/components.go` file. The component must be enabled
  only after sufficient testing and only when it meets [`Alpha` stability requirements](https://github.com/open-telemetry/opentelemetry-collector#alpha).

Reply:
Thank you, it has been removed.